e2e/cypress/tests/integration/custom_status/custom_status_1_spec.ts

```diff
@@ -10,13 +10,18 @@
 // Stage: @prod
 // Group: @custom_status
 
+import {set} from 'lodash';
+
 describe('Custom Status - CTAs for New Users', () => {
     before(() => {
-        cy.apiUpdateConfig({TeamSettings: {EnableCustomUserStatuses: true}});
-
-        // # Login as test user and visit channel
-        cy.apiInitSetup({loginAfter: true}).then(({team, channel}) => {
-            cy.visit(`/${team.name}/channels/${channel.name}`);
+        cy.apiGetConfig().then((config) => {
```

Comment:
Tests failed since it should be:
```diff
- cy.apiGetConfig().then((config) => {
+ cy.apiGetConfig().then(({config}) => {
```

Reply:
Thanks for pointing that out ... I did adjust the return value definition for this function, since it was incorrect (hence the error on my side) 👍🏼